internal/network/manifests_generator.go

```diff
@@ -274,3 +275,68 @@ func fillTemplate(manifestParams map[string]string, templateData string, log log
 	}
 	return buf.Bytes(), nil
 }
+
+const disableTunnelOffloadManifest = `
```

Comment:
why dont u reuse same manifests as were pushed to OCP?

https://github.com/openshift/machine-config-operator/blob/master/templates/common/vsphere/files/vsphere-disable-vmxnet3v4-features.yaml

there was an issue with nmcli afaik that's why they moved to ethtool. Lets not reinvent anything. Their path is tested and supported

Reply:
https://github.com/openshift/machine-config-operator/pull/2475#issuecomment-804888455

Reply:
@romfreiman This PR was tested before submit.
In general the correct way is to use nmcli. Please look at https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_networking/configuring-ethtool-offload-features_configuring-and-managing-networking for more details.
The problem with https://github.com/openshift/machine-config-operator/pull/2475#issuecomment-804888455  is that  _**nmcli connection up ...**_ was missing.
The difference is that this setting is managed by NM which means that if this setting is changed from some reason, NM returns it to the correct value (tested it).
Besides that it can be changed to match exactly https://github.com/openshift/machine-config-operator/blob/master/templates/common/vsphere/files/vsphere-disable-vmxnet3v4-features.yaml . I think it will require additional test cycle.

Reply:
@ori-amizur I think it is better to you use the manifest used by openshift. This way if we have problems in the future, we can have them handle it

Reply:
Lets push this in as is so that the we can release it, and in a different commit move it to use the openshift manifest